compose/ui/ui/src/webCommonW3C/kotlin/androidx/compose/ui/platform/WebImeInputService.kt

```diff
@@ -37,7 +37,9 @@ internal class WebImeInputService(parentInputService: InputAwareInputService) :
         onEditCommand: (List<EditCommand>) -> Unit,
         onImeActionPerformed: (ImeAction) -> Unit
     ) {
-        backingTextArea = BackingTextArea(imeOptions, onEditCommand, onImeActionPerformed)
+        backingTextArea = BackingTextArea(imeOptions, onEditCommand, onImeActionPerformed) { evt ->
+            sendKeyEvent(evt)
```

Comment:
Since this receives multiple callbacks, I'd prefer named parameters instead of trailing lambda syntax